src/datasources/floodscan.py

```diff
@@ -0,0 +1,82 @@
+import os
+from pathlib import Path
+
+import rioxarray as rxr
+import xarray as xr
+
+from src.utils import cloud_utils, cog_utils, gen_utils
+
+# from src.utils import blob
+
+DATA_DIR_GDRIVE = Path(os.getenv("AA_DATA_DIR_NEW"))
+FP_FS_HISTORICAL = (
+    DATA_DIR_GDRIVE
+    / "private"
+    / "raw"
+    / "glb"
+    / "FloodScan"
+    / "SFED"
+    / "SFED_historical"
+    / "aer_sfed_area_300s_19980112_20231231_v05r01.nc"
+)
+
+
+def open_historical_floodscan():
+    chunks = {"lat": 1080, "lon": 1080, "time": 1}
+    ds = xr.open_dataset(FP_FS_HISTORICAL, chunks=chunks)
+    da = ds["SFED_AREA"]
+    da = da.rio.set_spatial_dims(x_dim="lon", y_dim="lat")
+    da = da.rio.write_crs(4326)
+    return da
+
+
+def load_floodscan_cogs(
+    start_date,
+    end_date,
+    mode="dev",
+    container_name="global",
+    prefix="raster/cogs/aer_area_300s",
+):
+    container_client = cloud_utils.get_container_client(
+        mode=mode, container_name=container_name
+    )
+    cogs_list = [
+        x.name
+        for x in container_client.list_blobs(name_starts_with=prefix)
+        if (gen_utils.extract_date(x.name).date() >= start_date)
+        & (gen_utils.extract_date(x.name).date() <= end_date)  # noqa
+    ]
+
+    das = []
+    for cog in cogs_list:
+        da_in = process_floodscan_cog(
```

Comment:
While working off the old COGs in `global/raster/cogs/aer...` you will probably have to check that you're reading the right band. I found that the order was not consistent (i.e. usually it was `("SFED", "MFED")`, but not always, so had to write [this](https://github.com/OCHA-DAP/ds-floodexposure-monitoring/blob/7e94941c08101d0e95cd23e505aac3bacaed4edc/src/datasources/floodscan.py#L58-L67) to handle it). Otherwise sometimes you will accidentally grab the `MFED`, but it won't be obvious because the band names don't automatically get read in, just the order of the bands.

Reply:
nice, i will implement something like this... would be nice if there was a more direct solution where you could subset by long_name `ds["sfed"]`

Reply:
good call -- wrote a little function for this: [b238072](https://github.com/OCHA-DAP/hdx-floodscan/pull/1/commits/b238072c7f4c0a677c9c0fba1b3cd029df4e0a96)